### PR TITLE
UHF-834 CKEditor link

### PR DESCRIPTION
**Pull request by khalima**
How to test (if you have no setup etc.):
- `composer create-project City-of-Helsinki/drupal-helfi-platform:dev-main hel-platform --no-interaction --repository https://repository.drupal.hel.ninja/`
- Switch to corresponding branches: 
   - `composer require drupal/hdbt_admin:dev-UHF-834_ckeditor_link && composer require drupal/hdbt:dev-UHF-834_ckeditor_link && composer require drupal/helfi_platform_config:dev-UHF-834_ckeditor_link`
- Run `make new`
- Go to https://hel-platform.docker.sh/fi/esimerkki-laskeutumissivusta
   - Check that the landing page has hero and that there is a cta button "Katso myös esimerkkisivu" on that hero
   - Edit the page and expand the hero paragraph.
      - Check that the cta button comes from ckeditor.
      - Double click or right click (edit link) on the button and select another style for the link. 
      - Save the link and save the node. 
      - Check that the style of the cta button has changed. 
- Go to https://hel-platform.docker.sh/fi/hero-taustakuva
   - Check that page has hero and that there is a cta button "Tästä pääset Helsingin kaupungin etusivulle" on that hero.
   - Edit the page and expand the hero paragraph.
      - Check that the cta button comes from its own link field.
      - Add another link and change the link style to something different than the current one.
      - Save the node and check that the style of the cta button has changed and the new button has appeared.
      - Edit the page again and switch the style of the whole hero. F.e. "Kuva oikealla". Save the page and check that the buttons added in "Taustakuva" style doesn't appear on this style.

Also check these PRs:
- https://github.com/City-of-Helsinki/drupal-hdbt/pull/51
- https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/45
- https://github.com/City-of-Helsinki/drupal-hdbt-admin/pull/21

### CODE REVIEW

File: conf/cmi/field.field.node.article.field_content.yml [hunk 1]
```diff
@@ -56,11 +54,8 @@ settings:
       image:
         enabled: true
         weight: 2
-      link:
-        enabled: true
-        weight: 6
       list_of_links:
-        enabled: false
+        enabled: true
```
Comment (teroelonen) [inline conf/cmi/field.field.node.article.field_content.yml:58]: Is this accident or intentional change?

File: conf/cmi/field.field.node.page.field_content.yml [hunk 2]
```diff
@@ -45,10 +43,10 @@ settings:
         enabled: false
       columns:
         enabled: true
-        weight: -13
+        weight: 3
       gallery:
         weight: 15
-        enabled: false
+        enabled: true
```
Comment (teroelonen) [inline conf/cmi/field.field.node.page.field_content.yml:49]: How about this?

File: conf/cmi/field.field.node.page.field_content.yml [hunk 3]
```diff
@@ -58,11 +56,8 @@ settings:
       image:
         enabled: true
         weight: 2
-      link:
-        enabled: true
-        weight: 6
       list_of_links:
-        enabled: false
+        enabled: true
```
Comment (teroelonen) [inline conf/cmi/field.field.node.page.field_content.yml:60]: Or this?

**Review by teroelonen — CHANGES_REQUESTED** [review]
I listed some questions and also the linkit suggestions for the link field did not work for me.

File: conf/cmi/field.field.node.article.field_content.yml (hunk 1 above)
Comment (khalima) [reply conf/cmi/field.field.node.article.field_content.yml:58]: Must've been accident. Will fix it.

File: conf/cmi/field.field.node.page.field_content.yml (hunk 2 above)
Comment (khalima) [reply conf/cmi/field.field.node.page.field_content.yml:49]: Must've been accident. Will fix it.

File: conf/cmi/field.field.node.page.field_content.yml (hunk 3 above)
Comment (khalima) [reply conf/cmi/field.field.node.page.field_content.yml:60]: These must've come from some old db settings. I don't remember changing them..

**Review by teroelonen — APPROVED** [review]
This is now good to go!

**Pull request merged**